junit-jupiter-engine/src/main/java/org/junit/jupiter/engine/descriptor/TestMethodTestDescriptor.java

```diff
@@ -160,6 +163,22 @@ private void invokeBeforeTestExecutionCallbacks(JupiterEngineExecutionContext co
 		}
 	}
 
+	private void resolveParametersAndInvokeTestMethod(JupiterEngineExecutionContext context,
+			DynamicTestExecutor dynamicTestExecutor, ThrowableCollector throwableCollector) {
+		ExtensionRegistry registry = context.getExtensionRegistry();
+		Object[] arguments = parameterResolver.resolveParameters(getTestMethod(), Optional.empty(),
```

Comment:
The test instance cannot be "empty". You'll have to retrieve that and pass it on like in `invokeTestMethod()`.